azure/templates/ers-int-tests.yml

```diff
@@ -0,0 +1,42 @@
+steps:
```

Comment:
Looks like there is duplication between this and ers-smoke.yml. Is there a better way to structure this so it can be re-used?

Reply:
I've wondered about this but I'm not sure.

Reply:
This has now been done.